core/modules/qdisp/QueryRequest.cc

```diff
@@ -33,6 +33,7 @@
 #include "qdisp/QueryRequest.h"
 
 // System headers
+#include <cstddef>
```

Comment:
Oh, if its used elsewhere in the file then let's keep the direct 
include, thanks -- I missed that!

On 09/21/2015 02:01 PM, Jacek Becla wrote:
>
> In core/modules/qdisp/QueryRequest.cc 
> <https://github.com/lsst/qserv/pull/133#discussion_r40025463>:
>
> > @@ -33,6 +33,7 @@
> >  #include "qdisp/QueryRequest.h"
> >
> >  // System headers
> > +#include <cstddef>
>
> I wondered if anyone would ask :). I added it when doing original 
> migration and then John touch these lines so it got removed during 
> rebasing. I figured since we are using nullptr in that file, so it'd 
> be good to keep the include, although clearly some other header is 
> bringing it. I am ok with removing that include if you think it is better.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/lsst/qserv/pull/133/files#r40025463>.
>